test/extended/fixtures/test-build.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
it's generally less tedious to define them in json than to build them up programatically, in my experience.  but certainly compiler validation is useful.